src/qt/utilitydialog.cpp

```diff
@@ -100,6 +100,7 @@ HelpMessageDialog::HelpMessageDialog(QWidget *parent, HelpMode helpMode) :
         strUsage += HelpMessageOpt("-resetguisettings", tr("Reset all settings changed in the GUI").toStdString());
         if (showDebug) {
             strUsage += HelpMessageOpt("-uiplatform", strprintf("Select platform to customize UI for (one of windows, macosx, other; default: %s)", BitcoinGUI::DEFAULT_UIPLATFORM));
+            strUsage += HelpMessageOpt("-debug-ui", "Enable UI debug mode which allows to see live stylesheet updates in the UI if its used together with -custom-css-dir. It also forces some widgets to show up which are usually only visible under certain circumstances. (default: false)");
```

Comment:
I think the below suggestion makes this help text more understandable. 
```suggestion
            strUsage += HelpMessageOpt("-debug-ui", "Automatically update the live stylesheet with changes made on disk, especially useful with -custom-css-dir. Force some widgets to show up which are usually only visible under certain circumstances. (default: false)");
```

Reply:
Either my english is missing something there or "the live stylesheet" just doesn't make a lot sense?

Reply:
@PastaPastaPasta are you good with the changes in 1287779?